src/components/DocumentDropzone/DetailedErrors/DetailedErrors.tsx

```diff
@@ -12,26 +10,11 @@ export const DetailedError: FunctionComponent<{ title: string; message: string }
   );
 };
 
-export const DetailedErrors: FunctionComponent<{ verificationStatus: VerificationFragment[] | null }> = ({
-  verificationStatus,
-}) => {
-  if (!verificationStatus) return null;
-
-  const errors = [];
-
-  try {
-    const { hashValid, issuedValid, identityValid, revokedValid } = interpretFragments(verificationStatus);
-
-    if (!hashValid) errors.push(TYPES.HASH);
-    if (!issuedValid || !revokedValid) errors.push(TYPES.ISSUED);
-    if (!identityValid) errors.push(TYPES.IDENTITY);
-  } catch (e) {
-    errors.push(TYPES.INVALID);
-  }
```

Comment:
im not a fan of colllapsing all the checks here into the sagas for verification errros.
instead we could just extend the current behavior here.
you could just invoke your error handling function here, since its the only place that will use it.

Reply:
but shouldn't the redux be a single point of truth where it stores the document, validated fragments and the errors if there is any?
If we do it here we are just doing validation 2 times which is very confusing and i dont think its a good idea cos firstly it is a waste of resources and secondly we are uploading only 1 document and the error is being shown in the same page.